app/assets/stylesheets/application.tailwind.css

```diff
@@ -48,4 +48,42 @@
   .footer { @apply
     top-full sticky 
   }
+
+  .flip {
+    perspective: 1000px;
+    background-color: transparent;
+  }
+
+  .flip-content {
+    position: relative;
+    width: 100%;
+    height: 100%;
+    text-align: center;
+    transition: all 1s;
+    transform-style: preserve-3d;
```

Comment:
On the next issue, can you try `overflow-y: scroll` if it would work?

Reply:
will do!